src/components/StudentProfileBox/StudentProfileBox.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
Also, the component is appearing vertically for me. Not sure if this is an issue on my end? 
![image](https://user-images.githubusercontent.com/57849152/161923691-3666745b-fb99-4028-b5b1-a50962dc4edd.png)